--- origsrc/scribus-1.3.3.12/configure.in.in	2008-06-09 14:56:53.000000000 -0500
+++ src/scribus-1.3.3.12/configure.in.in	2008-06-15 00:04:53.421875000 -0500
@@ -587,18 +587,29 @@
 fi
 AC_SUBST(COREFOUNDATION_LIBS)
 
-dnl Some platforms require special libtool flags to build dynamically loadable
-dnl modules or plugins correctly. While libtool should take care of this with
-dnl a fixed set of arguments, in case we need to change it later this is being
-dnl made a configure-time substitution.
-MODULE_LINK_FLAGS=" -module "
-AC_SUBST(MODULE_LINK_FLAGS)
+AC_MSG_CHECKING([for some Win32 platform])
+case "$host" in
+	*-*-cygwin*|*-*-mingw*) platform_win32=yes ;;
+	*) platform_win32=no ;;
+esac
+AC_MSG_RESULT([$platform_win32])
+AM_CONDITIONAL(PLATFORM_WIN32, test "$platform_win32" = "yes")
 
 all_libraries="$all_libraries $USER_LDFLAGS"
 all_includes="$all_includes $USER_INCLUDES"
 AC_SUBST(all_includes)
 AC_SUBST(all_libraries)
 
+dnl Some platforms require special libtool flags to build dynamically loadable
+dnl modules or plugins correctly. While libtool should take care of this with
+dnl a fixed set of arguments, in case we need to change it later this is being
+dnl made a configure-time substitution.
+MODULE_LINK_FLAGS="-module -avoid-version \$(all_libraries) \$(LIB_QT)"
+if test "$platform_win32" = "yes" ; then
+	MODULE_LINK_FLAGS="$MODULE_LINK_FLAGS -no-undefined -Wl,\$(top_builddir)/scribus/libscribus.a"
+fi
+AC_SUBST(MODULE_LINK_FLAGS)
+
 AH_TOP([
 /*
  * Prevent code from including config.h directly. You should include
--- origsrc/scribus-1.3.3.12/scribus/Makefile.am	2008-02-26 13:48:49.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/Makefile.am	2008-06-15 00:04:53.437500000 -0500
@@ -436,6 +436,9 @@
 	$(INSTALL_DATA) $(srcdir)/vgradient.h                   $(prefix)/include/scribus/vgradient.h
 	$(INSTALL_DATA) $(srcdir)/vruler.h                      $(prefix)/include/scribus/vruler.h
 	$(INSTALL_DATA) $(srcdir)/werktoolb.h                   $(prefix)/include/scribus/werktoolb.h
+if PLATFORM_WIN32
+	$(INSTALL_DATA) libscribus.a $(libdir)
+endif
 
 uninstall-local:
 	-rm -f $(prefix)/include/scribus/about.h
@@ -627,10 +630,14 @@
 	-rm -f $(prefix)/include/scribus/vgradient.h
 	-rm -f $(prefix)/include/scribus/vruler.h
 	-rm -f $(prefix)/include/scribus/werktoolb.h
+if PLATFORM_WIN32
+	$(INSTALL_DATA) $(libdir)/libscribus.a
+endif
 
-scribus_LDADD  = $(LIBCAIRO_LIBS) $(LIBJPEG) $(LIBART_LIBS) $(LIBFREETYPE_LIBS) $(LIBFONTCONFIG_LIBS) $(LIBCUPS) $(LIBLCMS) $(LIBDL) $(LIBDL2) $(LIBTIFF) -lm $(LIB_QT) $(LIB_X11) $(LIBSOCKET) $(renderer_libs) text/libtext.la $(COREFOUNDATION_LIBS)
+scribus_LDADD  = text/libtext.la $(LIBCAIRO_LIBS) $(LIBJPEG) $(LIBART_LIBS) $(LIBFREETYPE_LIBS) $(LIBFONTCONFIG_LIBS) $(LIBCUPS) $(LIBLCMS) $(LIBDL) $(LIBDL2) $(LIBTIFF) -lm $(LIB_QT) $(LIB_X11) $(LIBSOCKET) $(renderer_libs) $(COREFOUNDATION_LIBS)
+
+AM_LDFLAGS = -export-dynamic
 
-AM_LDFLAGS = -export-dynamic $(LIBFREETYPE_LIBS)
 # set the include path for X, qt and KDE
 INCLUDES= $(LIBCAIRO_CFLAGS) $(LIBART_CFLAGS) $(LIBFREETYPE_CFLAGS) $(LIBFONTCONFIG_CFLAGS) $(all_includes)
 
@@ -639,6 +646,10 @@
 # the library search path.
 scribus_LDFLAGS = $(all_libraries)
 
+if PLATFORM_WIN32
+scribus_LDFLAGS += -Wl,--export-all-symbols,--out-implib,libscribus.a
+endif
+
 # them while "make clean", use CLEANFILES
 DISTCLEANFILES = $(scribus_METASOURCES)
 # this is to enable Qt behavoir of am_edit
--- origsrc/scribus-1.3.3.12/scribus/plugins/scriptplugin/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/scriptplugin/Makefile.am	2008-06-15 03:18:02.906250000 -0500
@@ -6,7 +6,7 @@
 pluginsdir = $(libdir)/scribus/plugins
 plugins_LTLIBRARIES = libscriptplugin.la
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
-libscriptplugin_la_LDFLAGS = -version-info 0:0:0 $(scripter2lib)
+libscriptplugin_la_LDFLAGS = -version-info 0:0:0 $(scripter2lib) $(LIBLCMS)
 libscriptplugin_la_METASOURCES = AUTO
 
 libscriptplugin_la_SOURCES = \
