--- origsrc/scribus-1.3.3.12/admin/acinclude.m4.in	2006-10-29 06:05:59.000000000 -0600
+++ src/scribus-1.3.3.12/admin/acinclude.m4.in	2008-06-15 03:02:59.218750000 -0500
@@ -4120,10 +4120,14 @@
 if test ! -r $python_libdir/libpython$version.so; then
   AC_FIND_FILE(libpython$version.a, $python_libdirs, python_libdir)
   if test ! -r $python_libdir/libpython$version.a; then
-    AC_FIND_FILE(python$version/config/libpython$version.a, $python_libdirs, python_libdir)
+   AC_FIND_FILE(python$version/config/libpython$version.dll.a, $python_libdirs, python_libdir)
     python_libdir=$python_libdir/python$version/config
-    if test ! -r $python_libdir/libpython$version.a; then
-      python_libdir=no
+   if test ! -r $python_libdir/libpython$version.dll.a; then
+     AC_FIND_FILE(python$version/config/libpython$version.a, $python_libdirs, python_libdir)
+     python_libdir=$python_libdir/python$version/config
+     if test ! -r $python_libdir/libpython$version.a; then
+       python_libdir=no
+     fi
     fi
   fi
 fi
