--- origsrc/scribus-1.3.6/CMakeLists.txt	2010-03-15 17:01:54.000000000 -0500
+++ src/scribus-1.3.6/CMakeLists.txt	2010-05-26 13:31:42.226786700 -0500
@@ -596,10 +596,13 @@ IF(NOT WIN32)
   IF (${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} LESS 2.5)
     SET(Cups_DIR ${CMAKE_MODULE_PATH})
   ENDIF (${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} LESS 2.5)
-  FIND_PACKAGE(Cups REQUIRED)
+  FIND_PACKAGE(Cups)
   IF(CUPS_FOUND)
     SET(HAVE_CUPS 1)
     MESSAGE("CUPS Library Found OK")
+  ELSE(CUPS_FOUND)
+    SET(CUPS_INCLUDE_DIR "")
+    SET(CUPS_LIBRARIES "")
   ENDIF(CUPS_FOUND)
 ELSE(NOT WIN32)
   # Windows doesn't use CUPS
--- origsrc/scribus-1.3.6/scribus/printdialog.cpp	2009-11-22 09:04:15.000000000 -0600
+++ src/scribus-1.3.6/scribus/printdialog.cpp	2010-05-26 11:05:20.876522200 -0500
@@ -24,7 +24,7 @@ for which a new license (GPL+exception)
 #if defined(_WIN32)
 	#include <windows.h>
 	#include <winspool.h>
-#else
+#elif defined(HAVE_CUPS)
 	#include <cups/cups.h>
 #endif
 #include "util_printer.h"
--- origsrc/scribus-1.3.6/scribus/util_printer.cpp	2007-10-24 18:09:51.000000000 -0500
+++ src/scribus-1.3.6/scribus/util_printer.cpp	2010-05-26 11:17:08.980023400 -0500
@@ -51,7 +51,7 @@ QStringList PrinterUtil::getPrinterNames
 		printerNames.sort();	
 	}
 	if ( printerInfos) free(printerInfos);
-#else
+#elif !defined(Q_OS_CYGWIN)
 	QString tmp;
 	QString Pcap;
 	QStringList wt;
