--- origsrc/scribus-1.3.3.12/scribus/plugins/barcodegenerator/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/barcodegenerator/Makefile.am	2008-06-15 00:04:53.562500000 -0500
@@ -15,7 +15,7 @@
 AM_LDFLAGS = $(LIBFREETYPE_LIBS) $(MODULE_LINK_FLAGS)
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus
+    -I$(top_srcdir)/scribus
 
 AM_CXXFLAGS = \
     $(GCC_CXX_VISIBILITY_FLAGS)
--- origsrc/scribus-1.3.3.12/scribus/plugins/colorwheel/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/colorwheel/Makefile.am	2008-06-15 00:04:53.578125000 -0500
@@ -15,7 +15,7 @@
 AM_LDFLAGS = $(LIBFREETYPE_LIBS) $(MODULE_LINK_FLAGS)
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus \
+    -I$(top_srcdir)/scribus \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
     -DSAMPLESDIR=\"$(datadir)/scribus/samples/\" \
--- origsrc/scribus-1.3.3.12/scribus/plugins/fileloader/oldscribusformat/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/fileloader/oldscribusformat/Makefile.am	2008-06-15 00:04:53.609375000 -0500
@@ -15,7 +15,7 @@
 AM_LDFLAGS = $(LIBFREETYPE_LIBS) $(MODULE_LINK_FLAGS)
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus
+    -I$(top_srcdir)/scribus
 
 AM_CXXFLAGS = \
     $(GCC_CXX_VISIBILITY_FLAGS)
--- origsrc/scribus-1.3.3.12/scribus/plugins/fileloader/oodraw/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/fileloader/oodraw/Makefile.am	2008-06-15 00:04:53.656250000 -0500
@@ -16,7 +16,7 @@
 AM_LDFLAGS = $(LIBFREETYPE_LIBS) $(MODULE_LINK_FLAGS) 
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus \
+    -I$(top_srcdir)/scribus \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
     -DSAMPLESDIR=\"$(datadir)/scribus/samples/\" \
--- origsrc/scribus-1.3.3.12/scribus/plugins/fontpreview/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/fontpreview/Makefile.am	2008-06-15 00:04:53.687500000 -0500
@@ -14,7 +14,7 @@
 AM_LDFLAGS = $(LIBFREETYPE_LIBS) $(MODULE_LINK_FLAGS)
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus \
+    -I$(top_srcdir)/scribus \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
     -DSAMPLESDIR=\"$(datadir)/scribus/samples/\" \
--- origsrc/scribus-1.3.3.12/scribus/plugins/gettext/csvim/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/gettext/csvim/Makefile.am	2008-06-15 00:04:53.718750000 -0500
@@ -16,7 +16,7 @@
 AM_LDFLAGS = $(LIBFREETYPE_LIBS) $(MODULE_LINK_FLAGS) 
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus \
+    -I$(top_srcdir)/scribus \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
     -DSAMPLESDIR=\"$(datadir)/scribus/samples/\" \
--- origsrc/scribus-1.3.3.12/scribus/plugins/gettext/docim/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/gettext/docim/Makefile.am	2008-06-15 00:04:53.921875000 -0500
@@ -16,7 +16,7 @@
 AM_LDFLAGS = $(LIBFREETYPE_LIBS) $(MODULE_LINK_FLAGS) 
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus \
+    -I$(top_srcdir)/scribus \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
     -DSAMPLESDIR=\"$(datadir)/scribus/samples/\" \
--- origsrc/scribus-1.3.3.12/scribus/plugins/gettext/htmlim/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/gettext/htmlim/Makefile.am	2008-06-15 00:04:53.921875000 -0500
@@ -16,7 +16,7 @@
 AM_LDFLAGS = $(LIBFREETYPE_LIBS) $(XML_LIBS) $(MODULE_LINK_FLAGS)
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus \
+    -I$(top_srcdir)/scribus \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
     -DSAMPLESDIR=\"$(datadir)/scribus/samples/\" \
--- origsrc/scribus-1.3.3.12/scribus/plugins/gettext/odtim/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/gettext/odtim/Makefile.am	2008-06-15 00:04:53.937500000 -0500
@@ -16,7 +16,7 @@
 AM_LDFLAGS = $(LIBFREETYPE_LIBS) $(XML_LIBS) $(MODULE_LINK_FLAGS)
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus \
+    -I$(top_srcdir)/scribus \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
     -DSAMPLESDIR=\"$(datadir)/scribus/samples/\" \
--- origsrc/scribus-1.3.3.12/scribus/plugins/gettext/pdbim/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/gettext/pdbim/Makefile.am	2008-06-15 00:04:54.125000000 -0500
@@ -16,7 +16,7 @@
 AM_LDFLAGS = $(LIBFREETYPE_LIBS) $(MODULE_LINK_FLAGS) 
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus \
+    -I$(top_srcdir)/scribus \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
     -DSAMPLESDIR=\"$(datadir)/scribus/samples/\" \
--- origsrc/scribus-1.3.3.12/scribus/plugins/gettext/sxwim/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/gettext/sxwim/Makefile.am	2008-06-15 00:04:54.156250000 -0500
@@ -16,7 +16,7 @@
 AM_LDFLAGS = $(LIBFREETYPE_LIBS) $(XML_LIBS) $(MODULE_LINK_FLAGS)
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus \
+    -I$(top_srcdir)/scribus \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
     -DSAMPLESDIR=\"$(datadir)/scribus/samples/\" \
--- origsrc/scribus-1.3.3.12/scribus/plugins/gettext/textfilter/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/gettext/textfilter/Makefile.am	2008-06-15 00:04:54.187500000 -0500
@@ -16,7 +16,7 @@
 AM_LDFLAGS = $(LIBFREETYPE_LIBS) $(MODULE_LINK_FLAGS) 
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus \
+    -I$(top_srcdir)/scribus \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
     -DSAMPLESDIR=\"$(datadir)/scribus/samples/\" \
--- origsrc/scribus-1.3.3.12/scribus/plugins/gettext/txtim/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/gettext/txtim/Makefile.am	2008-06-15 00:04:54.250000000 -0500
@@ -16,7 +16,7 @@
 AM_LDFLAGS = $(LIBFREETYPE_LIBS) $(MODULE_LINK_FLAGS) 
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus \
+    -I$(top_srcdir)/scribus \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
     -DSAMPLESDIR=\"$(datadir)/scribus/samples/\" \
--- origsrc/scribus-1.3.3.12/scribus/plugins/myplugin/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/myplugin/Makefile.am	2008-06-15 00:04:54.250000000 -0500
@@ -20,7 +20,7 @@
 AM_LDFLAGS = $(LIBFREETYPE_LIBS) $(MODULE_LINK_FLAGS)
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus
+    -I$(top_srcdir)/scribus
 
 AM_CXXFLAGS = \
     $(GCC_CXX_VISIBILITY_FLAGS)
--- origsrc/scribus-1.3.3.12/scribus/plugins/newfromtemplateplugin/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/newfromtemplateplugin/Makefile.am	2008-06-15 00:04:54.265625000 -0500
@@ -17,7 +17,7 @@
 AM_LDFLAGS = $(LIBFREETYPE_LIBS) $(MODULE_LINK_FLAGS)
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus \
+    -I$(top_srcdir)/scribus \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
     -DSAMPLESDIR=\"$(datadir)/scribus/samples/\" \
--- origsrc/scribus-1.3.3.12/scribus/plugins/pixmapexport/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/pixmapexport/Makefile.am	2008-06-15 00:04:54.312500000 -0500
@@ -16,7 +16,7 @@
 AM_LDFLAGS = $(LIBFREETYPE_LIBS) $(MODULE_LINK_FLAGS) 
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus \
+    -I$(top_srcdir)/scribus \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
     -DSAMPLESDIR=\"$(datadir)/scribus/samples/\" \
--- origsrc/scribus-1.3.3.12/scribus/plugins/psimport/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/psimport/Makefile.am	2008-06-15 00:04:54.359375000 -0500
@@ -24,6 +24,7 @@
 
 AM_CPPFLAGS = \
     -I$(top_builddir)/scribus \
+    -I$(top_srcdir)/scribus \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
--- origsrc/scribus-1.3.3.12/scribus/plugins/saveastemplateplugin/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/saveastemplateplugin/Makefile.am	2008-06-15 00:04:54.375000000 -0500
@@ -16,7 +16,7 @@
 AM_LDFLAGS = $(LIBFREETYPE_LIBS) $(MODULE_LINK_FLAGS) 
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus \
+    -I$(top_srcdir)/scribus \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
     -DSAMPLESDIR=\"$(datadir)/scribus/samples/\" \
--- origsrc/scribus-1.3.3.12/scribus/plugins/scriptplugin/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/scriptplugin/Makefile.am	2008-06-15 03:18:02.906250000 -0500
@@ -65,7 +65,7 @@
 AM_LDFLAGS = $(PYTHONLIB) $(LIBPYTHON) $(MODULE_LINK_FLAGS) $(BOOST_PYTHON_LDFLAGS)
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus $(PYTHONINC) \
+    -I$(top_srcdir)/scribus $(PYTHONINC) \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
     -DSAMPLESDIR=\"$(datadir)/scribus/samples/\" \
--- origsrc/scribus-1.3.3.12/scribus/plugins/scriptplugin/scripter2/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/scriptplugin/scripter2/Makefile.am	2008-06-15 00:04:54.453125000 -0500
@@ -50,8 +50,8 @@
 AM_LDFLAGS = $(PYTHONLIB) $(LIBPYTHON) $(MODULE_LINK_FLAGS) $(BOOST_PYTHON_LDFLAGS)
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus $(PYTHONINC) \
-    -I$(top_builddir)/scribus/plugins/scriptplugin \
+    -I$(top_srcdir)/scribus $(PYTHONINC) \
+    -I$(top_srcdir)/scribus/plugins/scriptplugin \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
     -DSAMPLESDIR=\"$(datadir)/scribus/samples/\" \
--- origsrc/scribus-1.3.3.12/scribus/plugins/short-words/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/short-words/Makefile.am	2008-06-15 00:04:54.484375000 -0500
@@ -15,7 +15,7 @@
 AM_LDFLAGS = $(LIBFREETYPE_LIBS) $(MODULE_LINK_FLAGS)
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus \
+    -I$(top_srcdir)/scribus \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
     -DSAMPLESDIR=\"$(datadir)/scribus/samples/\" \
--- origsrc/scribus-1.3.3.12/scribus/plugins/svgexplugin/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/svgexplugin/Makefile.am	2008-06-15 00:04:54.515625000 -0500
@@ -9,7 +9,7 @@
 EXTRA_DIST = svgexplugin.cpp svgexplugin.h  CMakeLists.txt
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus \
+    -I$(top_srcdir)/scribus \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
     -DSAMPLESDIR=\"$(datadir)/scribus/samples/\" \
--- origsrc/scribus-1.3.3.12/scribus/plugins/svgimplugin/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/plugins/svgimplugin/Makefile.am	2008-06-15 00:04:54.546875000 -0500
@@ -9,7 +9,7 @@
 EXTRA_DIST = color.h svgplugin.cpp svgplugin.h CMakeLists.txt
 
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus \
+    -I$(top_srcdir)/scribus \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
     -DSAMPLESDIR=\"$(datadir)/scribus/samples/\" \
--- origsrc/scribus-1.3.3.12/scribus/text/Makefile.am	2006-12-09 16:31:24.000000000 -0600
+++ src/scribus-1.3.3.12/scribus/text/Makefile.am	2008-06-15 00:04:54.562500000 -0500
@@ -9,10 +9,8 @@
 
 KDE_OPTIONS = qtonly
 
-AM_LDFLAGS = $(LIBFREETYPE_LIBS) $(MODULE_LINK_FLAGS)
-
 AM_CPPFLAGS = \
-    -I$(top_builddir)/scribus \
+    -I$(top_srcdir)/scribus \
     -DDOCDIR=\"$(datadir)/doc/scribus-$(VERSION)/\" \
     -DICONDIR=\"$(datadir)/scribus/icons/\" \
     -DSAMPLESDIR=\"$(datadir)/scribus/samples/\" \
