--- origsrc/scribus-1.4.4/CMakeLists.txt	2014-05-22 16:19:13.000000000 -0500
+++ src/scribus-1.4.4/CMakeLists.txt	2014-12-16 13:07:07.382431400 -0600
@@ -616,7 +616,7 @@ IF(NOT WIN32 AND NOT HAIKU)
   IF (${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} LESS 2.5)
     SET(Cups_DIR ${CMAKE_MODULE_PATH})
   ENDIF (${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} LESS 2.5)
-  FIND_PACKAGE(Cups REQUIRED)
+  FIND_PACKAGE(Cups)
   IF(CUPS_FOUND)
     SET(HAVE_CUPS 1)
     MESSAGE("CUPS Library Found OK")
@@ -780,11 +780,11 @@ INCLUDE_DIRECTORIES(
   ${ZLIB_INCLUDE_DIR}
 )
 
-IF(NOT WIN32 AND NOT HAIKU)
+IF(CUPS_FOUND)
   INCLUDE_DIRECTORIES(
     ${CUPS_INCLUDE_DIR}
   )
-ENDIF(NOT WIN32 AND NOT HAIKU)
+ENDIF(CUPS_FOUND)
 
 ##############################################################################################################
 ########## Uninstall Setup                                                                          ##########
--- origsrc/scribus-1.4.4/scribus/CMakeLists.txt	2014-12-16 12:54:15.385269300 -0600
+++ src/scribus-1.4.4/scribus/CMakeLists.txt	2014-12-16 13:06:45.162400700 -0600
@@ -782,11 +782,11 @@ TARGET_LINK_LIBRARIES(${EXE_NAME}
   ${WIN32_ONLY_LIBS}
 )
 
-IF(NOT WIN32 AND NOT HAIKU)
+IF(CUPS_FOUND)
   TARGET_LINK_LIBRARIES(${EXE_NAME}
     ${CUPS_LIBRARIES}
   )
-ENDIF(NOT WIN32 AND NOT HAIKU)
+ENDIF(CUPS_FOUND)
 
 # required to resolve symbols from libdl when ld is called with --no-add-needed
 # #9310 for the OpenBSD addition
