--- origsrc/scribus-1.4.4/scribus/scclocale.cpp	2013-02-24 15:48:22.000000000 -0600
+++ src/scribus-1.4.4/scribus/scclocale.cpp	2014-12-16 12:51:33.949040600 -0600
@@ -24,7 +24,7 @@ ScCLocale::ScCLocale()
 #if defined(Q_WS_WIN)
 	cLocale = _create_locale(LC_ALL, "C");
 #else
-  #if not defined(Q_OS_SOLARIS) and not defined(Q_OS_OPENBSD) and not defined (Q_OS_FREEBSD) and not defined (Q_OS_HAIKU)
+  #if not defined(Q_OS_SOLARIS) and not defined(Q_OS_OPENBSD) and not defined (Q_OS_FREEBSD) and not defined (Q_OS_HAIKU) and not defined (Q_OS_CYGWIN)
 	cLocale = newlocale(LC_ALL_MASK, "C", NULL);
   #endif
 #endif
@@ -35,7 +35,7 @@ ScCLocale::~ScCLocale()
 #if defined(Q_WS_WIN)
 	_free_locale(cLocale);
 #else
-  #if not defined(Q_OS_SOLARIS) and not defined(Q_OS_OPENBSD) and not defined (Q_OS_FREEBSD) and not defined (Q_OS_HAIKU)
+  #if not defined(Q_OS_SOLARIS) and not defined(Q_OS_OPENBSD) and not defined (Q_OS_FREEBSD) and not defined (Q_OS_HAIKU) and not defined (Q_OS_CYGWIN)
 	freelocale(cLocale);
   #endif
 #endif
@@ -110,7 +110,7 @@ double ScCLocale::strtod ( const char *
 #if defined(Q_WS_WIN)
 		return _strtod_l(str, endptr, that()->cLocale);
 #else
-  #if defined(Q_OS_SOLARIS) or defined (Q_OS_OPENBSD) or defined (Q_OS_FREEBSD) or defined (Q_OS_HAIKU)
+  #if defined(Q_OS_SOLARIS) or defined (Q_OS_OPENBSD) or defined (Q_OS_FREEBSD) or defined (Q_OS_HAIKU) or defined (Q_OS_CYGWIN)
 		char *oldlocale=setlocale(LC_NUMERIC, NULL);
 		double result(0.0);
 		setlocale(LC_NUMERIC, "C");
--- origsrc/scribus-1.4.4/scribus/scclocale.h	2013-02-24 15:43:32.000000000 -0600
+++ src/scribus-1.4.4/scribus/scclocale.h	2014-12-16 12:52:32.349121200 -0600
@@ -21,14 +21,14 @@
 #include <xlocale.h>
 #endif
 
-#if defined(Q_OS_SOLARIS) || defined (Q_OS_OPENBSD) || defined (Q_OS_FREEBSD) || defined (Q_OS_HAIKU)
+#if defined(Q_OS_SOLARIS) || defined (Q_OS_OPENBSD) || defined (Q_OS_FREEBSD) || defined (Q_OS_HAIKU) || defined (Q_OS_CYGWIN)
 #include <locale.h>
 #endif
 
 #if defined(Q_WS_WIN)
 #define XLocaleType _locale_t
 #else
-  #if defined (Q_OS_SOLARIS) || defined (Q_OS_OPENBSD) || defined (Q_OS_FREEBSD) || defined (Q_OS_HAIKU)
+  #if defined (Q_OS_SOLARIS) || defined (Q_OS_OPENBSD) || defined (Q_OS_FREEBSD) || defined (Q_OS_HAIKU) || defined (Q_OS_CYGWIN)
   #define XLocaleType char*   //dummy?
   #else
   #define XLocaleType locale_t
