--- origsrc/scribus-1.4.6/scribus/scclocale.cpp	2013-02-24 15:48:22.000000000 -0600
+++ src/scribus-1.4.6/scribus/scclocale.cpp	2017-07-24 00:06:46.567591800 -0500
@@ -10,6 +10,7 @@
 //
 //
 
+#define _GNU_SOURCE	// strtod_l
 #include "scclocale.h"
 
 #include <cstdlib>
--- origsrc/scribus-1.4.6/scribus/sccolorengine.cpp	2015-09-06 17:18:42.000000000 -0500
+++ src/scribus-1.4.6/scribus/sccolorengine.cpp	2017-07-24 00:03:34.752546300 -0500
@@ -21,6 +21,7 @@ for which a new license (GPL+exception)
  *                                                                         *
  ***************************************************************************/
 
+#define _GNU_SOURCE	// strtod_l
 #include "sccolorengine.h"
 #include "scribuscore.h"
 #include "colormgmt/sccolormgmtengine.h"
