--- origsrc/scribus-1.4.6/CMakeLists.txt	2017-07-23 23:46:37.435236700 -0500
+++ src/scribus-1.4.6/CMakeLists.txt	2017-07-24 00:10:11.432799400 -0500
@@ -524,14 +524,14 @@ ELSE(APPLE)
   ENDIF(${CMAKE_GENERATOR} MATCHES "^(Visual Studio|NMake).*")
 ENDIF(APPLE)
 
-IF(${CMAKE_GENERATOR} MATCHES "^(Visual Studio|NMake).*")
+IF(${CMAKE_GENERATOR} MATCHES "^(Visual Studio|NMake).*" OR CYGWIN)
   SET(CMAKE_CXX_FLAGS_RELEASE)
   SET(CMAKE_C_FLAGS_RELEASE)
-ELSE(${CMAKE_GENERATOR} MATCHES "^(Visual Studio|NMake).*")
+ELSE(${CMAKE_GENERATOR} MATCHES "^(Visual Studio|NMake).*" OR CYGWIN)
   SET(CMAKE_CXX_FLAGS_RELEASE "-O2 -Wall -fPIC")
   SET(CMAKE_C_FLAGS_RELEASE "-O2 -Wall -fPIC")
   ADD_DEFINITIONS(-DNDEBUG)
-ENDIF(${CMAKE_GENERATOR} MATCHES "^(Visual Studio|NMake).*")
+ENDIF(${CMAKE_GENERATOR} MATCHES "^(Visual Studio|NMake).*" OR CYGWIN)
 
 IF(WIN32)
   ADD_DEFINITIONS(-DWIN32_LEAN_AND_MEAN
--- origsrc/scribus-1.4.6/scribus/plugins/tools/2geomtools/lib2geom/CMakeLists.txt	2015-05-18 15:10:48.000000000 -0500
+++ src/scribus-1.4.6/scribus/plugins/tools/2geomtools/lib2geom/CMakeLists.txt	2017-07-24 00:10:45.188651400 -0500
@@ -18,9 +18,11 @@ SET(2GEOM_VERSION 0.1.0)
 # http://www.gentoo.org/proj/en/base/amd64/howtos/index.xml?part=1&chap=3 and
 # http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=559133
 
+IF (NOT CYGWIN)
 IF (ARCH_X86_64 EQUAL 1 OR ARCH_HPPA EQUAL 1 OR ARCH_MIPS EQUAL 1 OR ARCH_MIPS_64 EQUAL 1 OR ARCH_PPC_64 EQUAL 1)
   SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
 ENDIF (ARCH_X86_64 EQUAL 1 OR ARCH_HPPA EQUAL 1 OR ARCH_MIPS EQUAL 1 OR ARCH_MIPS_64 EQUAL 1 OR ARCH_PPC_64 EQUAL 1)
+ENDIF (NOT CYGWIN)
 
 OPTION(2GEOM_BUILD_SHARED
   "Build lib2geom and libtoy as shared libraries."
